### PR TITLE
Added Animated Unsubscribe Page

### PR DESCRIPTION
**Pull request by Shivangi-ch**
## Related Issue
  - Added Animated Unsubscribe Page

Closes: #1500 

### Describe the changes you've made
Added Animated Unsubscribe Page

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA
### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
unsub_page

### CODE REVIEW

**Comment by Shivangi-ch** [comment]
@ridsuteri @991rajat  Please review.

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@Shivangi-ch the project doesn't work on my side :( , i.e nothing happens on clicking the buttons also please fix the image in the project card

File: script.js [hunk 1]
```diff
@@ -69,6 +69,11 @@ let projectData = [
     projectImage: "assets/Images/bbc-news.png",
     projectUrl: "public/bbc-news-feed.html",
   },
+  {
+    projectName: "Animated Unsubscribe Page",
+    projectImage: "assets/Images/unsub_page.png",
```
Comment (991rajat) [inline script.js:74]: Link is not correct.
image

**Review by 991rajat — COMMENTED** [review]
Image is not loading. Fix that please.

**Comment by Shivangi-ch** [comment]
@991rajat Issue Fixed.

**Pull request merged**